### ISSUE TITLE
Compilation fails when boot2docker is not launched when using this plugin

### ISSUE THREAD

**Issue by ogirardot** [stebourbi/sbt-docker-launcher#5]
When a project uses this plugin on MacOS X or any OS using boot2docker, 
the compilation will fail if boot2docker is not up, instead of just failing at the precise "phase" (ex. it-test) using docker.

**Comment by ogirardot** [comment]
Exemple :

```
error in run: VM "boot2docker-vm" is not running. (Did you run `boot2docker up`?)
java.lang.RuntimeException: Nonzero exit code: 1
at scala.sys.process.BasicIO$Streamed$.scala$sys$process$BasicIO$Streamed$$next$1(BasicIO.scala:48)
at scala.sys.process.BasicIO$Streamed$$anonfun$apply$3.apply(BasicIO.scala:51)
at scala.sys.process.BasicIO$Streamed$$anonfun$apply$3.apply(BasicIO.scala:51)
at scala.sys.process.ProcessBuilderImpl$AbstractBuilder.lines(ProcessBuilderImpl.scala:143)
at scala.sys.process.ProcessBuilderImpl$AbstractBuilder.lines(ProcessBuilderImpl.scala:106)
at stebourbi.docker.launcher.DockerInfo$.createDockerInfo(DockerInfo.scala:32)
at stebourbi.docker.launcher.DockerInfo$.dockerInfo$lzycompute(DockerInfo.scala:47)
at stebourbi.docker.launcher.DockerInfo$.dockerInfo(DockerInfo.scala:47)
at InfrastructureContainer$.createContainerCassandra$default$3(InfrastructureContainer.scala:23)
at Build$$anonfun$root$8.apply(Build.scala:12)
at Build$$anonfun$root$8.apply(Build.scala:12)
at sbt.EvaluateSettings$$anonfun$sbt$EvaluateSettings$$constant$1.apply(INode.scala:163)
at sbt.EvaluateSettings$$anonfun$sbt$EvaluateSettings$$constant$1.apply(INode.scala:163)
at sbt.EvaluateSettings$MixedNode.evaluate0(INode.scala:175)
at sbt.EvaluateSettings$INode.evaluate(INode.scala:135)
at sbt.EvaluateSettings$$anonfun$sbt$EvaluateSettings$$submitEvaluate$1.apply$mcV$sp(INode.scala:69)
at sbt.EvaluateSettings.sbt$EvaluateSettings$$run0(INode.scala:78)
at sbt.EvaluateSettings$$anon$3.run(INode.scala:74)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
at java.lang.Thread.run(Thread.java:745)
    [error] Nonzero exit code: 1
```

**Comment by stebourbi** [comment]
fixed with #7
